source/guide_seafile.rst

```diff
@@ -0,0 +1,271 @@
+
+Configure port
+--------------
+
+Since seafile uses its own webserver and fileserver, you need to find 2 free ports. Run this command 2 times and write down the 2 ports.
+
+.. include:: includes/generate-port.rst
+
+Step 5
+------
+
+Enter your domain name in config; Edit ``~/seafile/conf/ccnet.conf``
+
+.. warning:: Replace ``isabell`` with your username!
+
+.. code-block:: console
+  :emphasize-lines: 1
+
+  SERVICE_URL = https://isabell.uber.space/
+
+Step 6
+------
+
+Change seahub port in config; Edit ``~/seafile/conf/gunicorn.conf``
+
+.. warning:: Replace ``<yourport>`` with your first port!
+
+.. code-block:: console
+  :emphasize-lines: 1
+
+  bind = "0.0.0.0:<yourport>"
+
+Step 7
+------
+
+Change seafile port in config; Edit ``~/seafile/conf/seafile.conf``
+
+.. warning:: Replace ``<yoursecondport>`` with your second port!
+
+.. code-block:: console
+  :emphasize-lines: 2
+
+  [fileserver]
+  port = <yoursecondport>
+
+Step 8
+------
+
+Change seahub config; Edit ``~/seafile/conf/seahub_settings.py``
+Add the following lines:
+
+.. warning:: Replace ``isabell`` with your username!
+
+.. code-block:: console
+  :emphasize-lines: 1,2,6,8
+
+  SITE_BASE = 'https://isabell.uber.space'
+  SITE_NAME = 'isabell.uber.space'
+
+  SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+
+  FILE_SERVER_ROOT = 'https://isabell.uber.space/seafhttp'
+
+  CSRF_TRUSTED_ORIGINS = ['isabell.uber.space']
+
+Step 9 - Setup .htaccess
+------------------------
+
+Create a ``~/html/.htaccess`` file with the following content:
+
+.. warning:: Replace ``<yourport>`` with your first port!
+.. warning:: Replace ``<yoursecondport>`` with your second port!
+
+.. code-block:: apache
+  :emphasize-lines: 6,7
+
+  RewriteEngine On
+  RewriteCond %{REQUEST_FILENAME} !-f
+  DirectoryIndex disabled
+  RewriteBase /
+
+  RewriteRule ^seafhttp/(.*) http://localhost:<yoursecondport>/$1 [P]
+  RewriteRule ^(.*)$ http://localhost:<yourport>/$1 [P]
+
+
+In our example this would be:
+
+.. code-block:: apache
+
+  RewriteEngine On
+  RewriteCond %{REQUEST_FILENAME} !-f
+  DirectoryIndex disabled
+  RewriteBase /
+
+  RewriteRule ^seafhttp/(.*) http://localhost:9001/$1 [P]
+  RewriteRule ^(.*)$ http://localhost:9000/$1 [P]
+
+
+Finishing installation
+======================
+
+Restart seafile and seahub
+--------------------------
+
+::
+
+ [isabell@stardust ~]$ cd ~/seafile/seafile-server-*
+ [isabell@stardust ~]$ ./seafile.sh restart
+ [isabell@stardust ~]$ ./seahub.sh restart
+ [isabell@stardust ~]$
+
+With starting seahub for the first time, you have to create an admin account.
+
+Now you can point your browser to your domain and login with your admin account.
+
+
+.. _Python: https://manual.uberspace.de/en/lang-python.html
+.. _MySQL: https://manual.uberspace.de/en/database-mysql.html
+.. _domains: https://manual.uberspace.de/en/web-domains.html
+.. _credentials: https://manual.uberspace.de/en/database-mysql.html#login-credentials
+
+----
```

Comment:
Can you please add a section explaining how to update an existing seafile instance? Thanks :)